extensions/lifecycle/LifecycleQueuePopulator.js

```diff
@@ -37,13 +37,24 @@ class LifecycleQueuePopulator extends QueuePopulatorExtension {
      */
     constructor(params) {
         super(params);
+        this._vaultAdmin = params.vaultAdmin;
         this._authConfig = params.authConfig;
 
+        let vaultAdminUseTempCreds = true;
+        // Falling back to the global lifecycle auth config when in S3C
+        // and disabling AssumeRole
+        if (!this._vaultAdmin) {
+            this._vaultAdmin = params.config.auth.vault;
+            this._authConfig = params.config.auth;
+            vaultAdminUseTempCreds = false;
+        }
```

Comment:
in relation with https://github.com/scality/backbeat/pull/2575/files#r1830765541 , would be nicer to have just a fallback in the config:

```suggestin
        this._vaultAdmin = params.vaultAdmin || params.config.auth.vault;
        this._authConfig = params.authConfig || params.config.auth;
```

...or maybe even better update the config in S3C (if we can?)